service/fsStorage.go

```diff
@@ -0,0 +1,92 @@
+package service
+
+import (
+	"awesomeProject3/storage"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+const filename = "test.json"
```

Comment:
Pass filename into `fsStorage` constructor. That way you could easily test it afterwards and provide a certain path for you storage.